src/app/tests/suites/commands/discovery/DiscoveryCommands.cpp

```diff
@@ -175,6 +175,17 @@ void DiscoveryCommands::OnNodeDiscovered(const chip::Dnssd::DiscoveredNodeData &
         data.mrpRetryIntervalActive.SetValue(nodeData.resolutionData.mrpRetryIntervalActive.Value().count());
     }
 
+    // TODO need to add new entries for kDefaultResponse in DiscoveryCommands.js (project-chip/zap) before adding this
```

Comment:
worth while to create an issue to track this?

Reply:
not really. It was already not updated and is part of the simulated cluster test. It is not really needed. I already have a PR for zap that I just need to test. I'll do it when I have some time